unit/src/TabulatedFluidPropertiesTest.C

```diff
@@ -120,3 +122,25 @@ TEST_F(TabulatedFluidPropertiesTest, fromFile)
   REL_TEST("cv", _tab_fp->cv(p, T), _co2_fp->cv(p, T), 1.0e-4);
   REL_TEST("entropy", _tab_fp->s(p, T), _co2_fp->s(p, T), 1.0e-4);
 }
+
+// Test generation of tabulated fluid properties
+TEST_F(TabulatedFluidPropertiesTest, generateTabulatedData)
+{
+  // We always want to generate a new file in this test, delete existing file
+  std::remove("fluid_properties.csv");
```

Comment:
You want to do this in `TearDown()`, see [advanced guide](https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md) to the unit testing package.

Reply:
I've moved this to `TearDown()`